dotnet/src/Experimental/Process.LocalRuntime/LocalProcess.cs

```diff
@@ -56,7 +57,7 @@ public LocalProcess(KernelProcess process, Kernel kernel, string? parentProcessI
     /// Loads the process and initializes the steps. Once this is complete the process can be started.
     /// </summary>
     /// <returns>A <see cref="Task"/></returns>
-    public async Task LoadAsync()
+    public ValueTask InitializeProcessAsync()
```

Comment:
Does this need to be public not that the initialization is done via `_initialiseTask`?

If it stays public how do we prevent initialization being performed multiple times?

Reply:
The class is internal, but your point is valid. Made this private.